scripts/requirements-extras.txt

```diff
@@ -29,3 +29,6 @@ PyGithub
 
 # used to generate devicetree dependency graphs
 graphviz
+
+# used to generate CBOR encoders and decoders
```

Comment:
To give a pointer to someone who has no idea what it is:
```suggestion
# used to generate CBOR encoders and decoders, e.g. lwm2m_senml_cbor.
```

Reply:
Ok. done.